Celeste.Mod.mm/Patches/Level.cs

```diff
@@ -635,6 +635,10 @@ static partial class MonoModRules {
                 }
             }
 
+            cctor_il.Emit(OpCodes.Dup);
+            cctor_il.Emit(OpCodes.Ldstr, "theoCrystalHoldingBarrier"); // theoCrystalHoldingBarrier is an unused entity appearing in chapter 5
+            cctor_il.Emit(OpCodes.Callvirt, m_LoadStrings_Add);
+            cctor_il.Emit(OpCodes.Pop);
```

Comment:
Feels weird to have this jammed into another patch, but this is one of those patches that needs to be refactored anyway so maybe it can just be a future problem™️ lol

Reply:
I considered just adding a separate check for it, but adding it to `_LoadStrings` seemed like the cleanest way, and I don't see a good place to do that other than the .cctor patch.